google-cloud-spanner/lib/google/cloud/spanner/convert.rb

```diff
@@ -164,14 +169,16 @@ def field_for_object obj
             else
               if obj.respond_to?(:read) && obj.respond_to?(:rewind)
                 :BYTES
+              elsif obj.class.respond_to?(:descriptor)
```

Comment:
It's probably simpler to check the type against `Google::Protobuf::MessageExts` (i.e. add it to the case statement) instead of putting it here.